pkg/products/grafana/reconciler.go

```diff
@@ -90,28 +63,26 @@ func NewReconciler(configManager config.ConfigReadWriter, installation *integrea
 		installation:  installation,
 		mpm:           mpm,
 		log:           logger,
-		Reconciler:    resources.NewReconciler(mpm).WithProductDeclaration(*productDeclaration),
+		Reconciler:    resources.NewReconciler(mpm),
 		recorder:      recorder,
 	}, nil
 }
 
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=list;get;watch;update;delete
+
 func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1alpha1.RHMI, productStatus *integreatlyv1alpha1.RHMIProductStatus, client k8sclient.Client, productConfig quota.ProductConfig, uninstall bool) (integreatlyv1alpha1.StatusPhase, error) {
 	r.log.Info("Start Grafana reconcile")
+	if resources.IsInProw(installation) {
+		return integreatlyv1alpha1.PhaseCompleted, nil
```

Comment:
could we add a log here - something like "Running in prow, skipping Grafana reconciliation"

Reply:
log added. Thank you